cmd/collector/app/builder/span_handler_builder.go

```diff
@@ -116,7 +116,7 @@ func (spanHb *SpanHandlerBuilder) initElasticStore(esBuilder escfg.ClientBuilder
 // BuildHandlers builds span handlers (Zipkin, Jaeger)
 func (spanHb *SpanHandlerBuilder) BuildHandlers() (app.ZipkinSpansHandler, app.JaegerBatchesHandler) {
 	hostname, _ := os.Hostname()
-	hostMetrics := spanHb.metricsFactory.Namespace(hostname, nil)
+	hostMetrics := spanHb.metricsFactory.Namespace("", map[string]string{"host": hostname})
```

Comment:
NOTE: this is a breaking change, but without it the crossdock test was failing with `"49d962b96482:save_latency" is not a valid metric name` because the host name was used as a prefix.